### PR TITLE
Allow optional sql field

### LINKED ISSUE #82: Allow for other incoming SQL fields
The Go SQL parser looks for these items

https://github.com/go-spatial/maptoolkit/blob/98544368ed5181ba5ea388170664c1cfc7af3b0f/atlante/grids/postgresql/postgresql.go#L296-L316

**Request** — allow these constraints for incoming SQL fields

* required — `mdg_id `, `sheet`,`series`,`geom` in [epsg.io/4326](epsg.io/4326)  only if *lat,*lng values are not given.
* inferred — coordinates such as `swlat` or `nelon`, etc
* optional — `country`, `city`, or _other administrative level_

### PR DESCRIPTION
**Pull request by gdey**
This allows the Postgres driver for grids to take optional fields.

fixes #82

### CODE REVIEW

File: atlante/grids/postgresql/postgresql.go [hunk 1]
```diff
@@ -3,12 +3,19 @@ package postgresql
...
 	"crypto/tls"
 	"crypto/x509"
-	"database/sql"
 	"fmt"
 	"io/ioutil"
+	"strconv"
 	"sync"
 	"time"
 
+<<<<<<< HEAD
```
Comment (ARolek) [inline atlante/grids/postgresql/postgresql.go]: left over merge conflict

File: atlante/grids/postgresql/postgresql.go [hunk 2]
```diff
@@ -86,8 +93,38 @@ const (
...
+	SQLSWLNGDMSField = "swlng_dms"
+	SQLNELATDMSField = "nelat_dms"
+	SQLNELNGDMSField = "nelng_dms"
+	SQLCountryField  = "country"
+	SQLCityField     = "city"
+	SQLEditedByField = "edited_by"
+	SQLEditedAtField = "edited_at"
 )
 
+<<<<<<< HEAD
```
Comment (ARolek) [inline atlante/grids/postgresql/postgresql.go]: left over merge conflict

File: atlante/grids/postgresql/postgresql.go (hunk 1 above)
Comment (gdey) [reply atlante/grids/postgresql/postgresql.go]: Sorry, seem I forgot to save the file.

File: atlante/grids/postgresql/postgresql.go (hunk 2 above)
Comment (gdey) [reply atlante/grids/postgresql/postgresql.go]: Sorry, seem I forgot to save the file.

**Pull request merged**